src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -111,6 +111,15 @@ class CosmasGoogleCloudService(
         }
     }
 
+    private fun getTTL(info: ProjectInfo): Long {
+        val day: Long = 24 * 60 * 60 * 1000
```

Comment:
extract this to toplevel constant? and rename to MILLIS_IN_DAY?

Reply:
done